docs/reference/sciencedirect/ScienceDirectSearch.rst

```diff
@@ -0,0 +1,146 @@
+pybliometrics.scopus.ScienceDirectSearch
+==========================================
+
+`ScopusSearch()` implements the `Science Direct Search API <https://nonprod-devportal.elsevier.com/documentation/ScienceDirectSearchAPI.wadl>`_.  It executes a query to search for documents and retrieves the resulting records.
+Any query that works in the `Advanced Document Search on sciencedirect.com <https://www.sciencedirect.com/search/entry>`_ will work.
+For a complete guide on how to query check the `documentation <https://service.elsevier.com/app/answers/detail/a_id/25974/supporthub/sciencedirect/>`_.
+
+.. currentmodule:: pybliometrics.sciencedirect
+.. contents:: Table of Contents
+    :local:
+
+Documentation
+-------------
+
+.. autoclass:: ScienceDirectSearch
+   :members:
+   :inherited-members:
+
+Examples
+--------
+
+The class is initialized with a search query. To see the download progress, set `verbose=True`.
+
+.. code-block:: python
+
+    >>> from pybliometrics.sciencedirect import ScienceDirectSearch, init
+    >>> init()
+    >>> # Retrieve documents based on the search query  
+    >>> sds = ScienceDirectSearch('"neural radiance fields" AND "3D" AND YEAR(2024)', verbose=True)
+    Downloading results for query ""neural radiance fields" AND "3D" AND YEAR(2024)":
+    88%|████████▊ | 7/8 [00:04<00:00,  1.24it/s]
+
+To access the results, use the attribute `results` which contains a list of `Document` namedtuples.
+
+.. code-block:: python
+
+    >>> # Access the results
+    >>> results = sds.results
```

Comment:
Demonstrate the length of the results, and show how it can be used to create a pandas Dataframe